components/org.wso2.carbon.appmgt.hostobjects/src/main/java/org/wso2/carbon/appmgt/hostobjects/APIStoreHostObject.java

```diff
@@ -1291,6 +1291,58 @@ public static NativeArray jsFunction_getAPI(Context cx, Scriptable thisObj,
         return myn;
     }
 
+    public static NativeArray jsFunction_getServerURL(Context cx, Scriptable thisObj,
+                                                Object[] args, Function funObj) throws ScriptException,
+                                                                                       AppManagementException {
+
+        String providerName;
+        String transport;
+
+        NativeArray myn = new NativeArray(0);
+        if (args!=null && args.length != 0) {
```

Comment:
Better check exact number of expected arguments and throw exception instead of (args!=null && args.length != 0) . Otherwise this can cause ArrayindexOutofBound exception.